docs/_posts/2015-10-07-react-v0.14.md

```diff
@@ -0,0 +1,201 @@
+
+    ```js
+    var React = require('react');
+    var ReactDOM = require('react-dom');
+
+    var MyComponent = React.createClass({
+      render: function() {
+        return <div>Hello World</div>;
+      }
+    });
+
+    ReactDOM.render(<MyComponent />, node);
+    ```
+
+    The old names will continue to work with a warning until 0.15 is released, and we’ve published the [automated codemod script](https://github.com/facebook/react/blob/master/packages/react-codemod/README.md) we used at Facebook to help you with this transition.
+
+    The add-ons have moved to separate packages as well:
+    - `react-addons-clone-with-props`
+    - `react-addons-create-fragment`
+    - `react-addons-css-transition-group`
+    - `react-addons-linked-state-mixin`
+    - `react-addons-perf`
+    - `react-addons-pure-render-mixin`
+    - `react-addons-shallow-compare`
+    - `react-addons-test-utils`
+    - `react-addons-transition-group`
+    - `react-addons-update`
+    - `ReactDOM.unstable_batchedUpdates` in `react-dom`.
+
+    For now, please use matching versions of `react` and `react-dom` (and the add-ons, if you use them) in your apps to avoid versioning problems.
+
+- #### DOM node refs
+
+    The other big change we’re making in this release is exposing refs to DOM components as the DOM node itself. That means: we looked at what you can do with a `ref` to a React DOM component and realized that the only useful thing you can do with it is call `this.refs.giraffe.getDOMNode()` to get the underlying DOM node. Starting with this release, `this.refs.giraffe` _is_ the actual DOM node. **Note that refs to custom (user-defined) components work exactly as before; only the built-in DOM components are affected by this change.**
+
+    ```js
+    var Zoo = React.createClass({
+      render: function() {
+        return <div>Giraffe name: <input ref="giraffe" /></div>;
+      },
+      showName: function() {
+        // Previously: var input = this.refs.giraffe.getDOMNode();
+        var input = this.refs.giraffe;
+        alert(input.value);
+      }
+    });
+    ```
+
+    This change also applies to the return result of `ReactDOM.render` when passing a DOM node as the top component. As with refs, this change does not affect custom components.
+    
+    With this change, we’re deprecating `.getDOMNode()` and replacing it with `ReactDOM.findDOMNode` (see below). If your components are currently using `.getDOMNode()`, they will continue to work with a warning until 0.15.
+
+- #### Stateless functional components
+
+    In idiomatic React code, most of the components you write will be stateless, simply composing other components. We’re introducing a new, simpler syntax for these components where you can take `props` as an argument and return the element you want to render:
+
+    ```js
+    // A functional component using an ES2015 (ES6) arrow function:
+    var Aquarium = (props) => {
```

Comment:
I'd be tempted to take out both these examples and let people see the docs.  But if we want to have examples, we should have at least one ES5 example since that's what browsers run.